zokrates_cli/src/ops/compute_witness.rs

```diff
@@ -38,6 +39,13 @@ pub fn subcommand() -> App<'static, 'static> {
         .takes_value(true)
         .required(false)
         .default_value(cli_constants::WITNESS_DEFAULT_PATH)
+    ).arg(Arg::with_name("witness")
```

Comment:
I think this should be called `circom-witness` or something similar, as we also output the witness.